lib/fpm/package/cpan.rb

```diff
@@ -25,6 +25,9 @@ class FPM::Package::CPAN < FPM::Package
   option "--test", :flag,
     "Run the tests before packaging?", :default => true
 
+  option "--verbose", :flag,
+    "Produce verbose output from cpanm?", :default => true
```

Comment:
Thoughts on this being false by default instead of true?

Reply:
oops that's a bug, stand by for a new pull request!

Reply:
(I was literally JUST fixing this as you made the comment, haha!)